src/prisma/schema.prisma

```diff
@@ -74,3 +78,40 @@ model Notification {
   authorId     Int
   author       Player   @relation(fields: [authorId], references: [id])
 }
+
+model Match {
+  id        Int         @id @default(autoincrement())
+  opponent  String
+  result    Result?
+  homeGoals Int?
```

Comment:
Vurdert å heller regne ut antall mål fra statistics og dermed også resultat istedenfor at det må legges inn i databasen?

Reply:
Ja vurderte det, men tenkte det var litt enklere å bare gjøre det sånn. Er litt hipp som happ for min del